x-pack/plugins/security_solution/public/cases/containers/use_get_case_user_actions.tsx

```diff
@@ -312,7 +312,7 @@ export const useGetCaseUserActions = (
 
   useEffect(() => {
     if (!isEmpty(caseId)) {
-      fetchCaseUserActions(caseId, subCaseId);
+      fetchCaseUserActions(caseId, caseConnectorId, subCaseId);
```

Comment:
Do we need to update the dependencies line above since the connector id is taken as a parameter now?


```
    },
    // eslint-disable-next-line react-hooks/exhaustive-deps
    [caseConnectorId]
  );
```